Sources/ArcGISToolkit/Documentation.docc/Resources/FeatureFormView/FeatureFormViewStep2.swift

```diff
@@ -0,0 +1,23 @@
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct FeatureFormExampleView: View {
+    static func makeMap() -> Map {
+        let portalItem = PortalItem(
+            portal: .arcGISOnline(connection: .anonymous),
+            id: Item.ID("9f3a674e998f461580006e626611f9ad")!
+        )
+        return Map(item: portalItem)
+    }
+    
+    @StateObject private var dataModel = MapDataModel(
+        map: makeMap()
+    )
+    
+    @State private var identifyScreenPoint: CGPoint?
+    
+    @State private var featureForm: FeatureForm?
+    
+    @State private var showFeatureForm = false
```

Comment:
Just a suggestion to match the "reads like an assertion" guideline. Take or leave.

```suggestion
    @State private var featureFormIsPresented = false
```

Reply:
Fixing in example in #653.